graphics/rcd/lang/de_DE.yml

```diff
@@ -41,22 +41,27 @@ strings:
 			one:   "Erreiche mindestens %1% Besucher im Park."
 			other: "Erreiche mindestens %1% Besucher im Park."
 		OBJECTIVETEXT_PARK_RATING: "Die Parkbewertung muss mindestens %1% betragen."
+		OBJECTIVETEXT_PARK_VALUE:  "Der Park-Verkehrswert muss mindestens %1% betragen."
 
 		NUMBERED_INSTANCE_NAME: "%1% #%2%"
 		RESOLUTION:             "%1%×%2%"
 		GUEST_NAME:             "Besucher %1%"
 
+		NR_DAYS:
+			one:                  "%1% Tag"
+			other:                "%1% Tage"
+		APPLY:                    "Anwenden"
```

Comment:
Middle 2 are somewhat off wrt indenting.

Reply:
Artefact of tabs for indent. When we finally move to Weblate (soon-ish) it will likely break all indenting again anyway and we will no longer maintain these files manually.